http-endpoint/src/main.rs

```diff
@@ -143,10 +144,11 @@ async fn main() -> anyhow::Result<()> {
     };
 
     HttpServer::new(move || {
-        let auth = HttpAuthentication::bearer(validator);
+        //let jwt_auth = HttpAuthentication::bearer(jwt_validator);
+        let basic_auth = HttpAuthentication::basic(basic_validator);
 
         App::new()
-            .wrap(Condition::new(enable_auth, auth))
+            .wrap(Condition::new(enable_auth, basic_auth))
```

Comment:
Btw, I figured out that you can scope this:

~~~rust
            .service(
                web::scope("/api/v1")
                    .wrap(Condition::new(enable_auth, auth))
                    .service(info::get_info)
                    .service(spy::stream_events),
            )
            .service(index)
~~~